src/lib/components/tag/index.tsx

```diff
@@ -0,0 +1,104 @@
+import stylin, { variant } from '@stylin.js/react';
+import { CustomDomComponent, easeInOut, motion } from 'framer-motion';
+import React, { FC, PropsWithChildren } from 'react';
+
+import { Box, Typography } from '../../elements';
+import { TimesSVG } from '../../icons';
+import { colors } from '../../theme/dark/colors';
+import { TagElementProps, TagProps } from './tag.types';
+
+const TagElement = stylin<TagElementProps>('button')(
+  variant({
+    scale: 'tags',
+    property: 'variant',
+  })
+);
+
+const MotionTag = motion(TagElement) as CustomDomComponent<TagProps>;
+
+export const Tag: FC<PropsWithChildren<TagProps>> = ({
+  onClose,
+  children,
+  PrefixIcon,
+  hasCloseIcon,
+  size = 'large',
+  ...props
+}) => {
+  return (
+    <MotionTag
+      {...(props.variant === 'filled' || props.variant === 'outline'
+        ? {
+            py:
+              size === 'large' ? '2xs' : size === 'medium' ? '2xs' : '.125rem',
```

Comment:
Is your intention to keep size 2xs as default? To skip this redundancy, you can do it this way:  `size ? '2xs' : size === 'medium' ? '2xs' : .125rem`. But note that the size will be 2xs as default

Reply:
@MarioBatalha The default size is 'Large' which is initialized in the props above